torchvision/utils.py

```diff
@@ -61,25 +66,28 @@ def make_grid(
 
     if normalize is True:
         tensor = tensor.clone()  # avoid modifying tensor in-place
-        if range is not None:
-            assert isinstance(range, tuple), \
-                "range has to be a tuple (min, max) if specified. min and max are numbers"
+        if "range" in kwargs.keys():
+            if kwargs["range"] is not None:
+                assert isinstance(kwargs["range"], tuple), \
+                    "range has to be a tuple (min, max) if specified. min and max are numbers"
+            vmin, vmax = get_range(kwargs["range"])
 
         def norm_ip(img, low, high):
             img.clamp_(min=low, max=high)
             img.sub_(low).div_(max(high - low, 1e-5))
 
-        def norm_range(t, range):
-            if range is not None:
-                norm_ip(t, range[0], range[1])
+        def norm_range(t, vmin, vmax):
+            if vmin is not None and vmax is not None:
+                norm_ip(t, vmin, vmax)
```

Comment:
What if one of them is not None? If it reasonable to ignore the value of the other?

Reply:
I did not consider it before, this is the reference 
> https://github.com/pytorch/vision/blob/master/torchvision/utils.py#L64. 

vmin and vmax was deprecated.